src/OpenTelemetry.Abstractions/Metrics/IGaugeLong.cs

```diff
@@ -0,0 +1,28 @@
+﻿// <copyright file="IGaugeLong.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Metrics
+{
+    using OpenTelemetry.Metrics.Implementation;
+
+    /// <summary>
+    /// Gauge metric, to report instantaneous measurement of a double value. Cumulative values can go
+    /// up or stay the same, but can never go down.Cumulative values cannot be negative.
+    /// </summary>
+    public interface IGaugeLong : IMetric<IGaugeLongTimeSeries>
+    {
```

Comment:
Java implementation is different

https://github.com/open-telemetry/opentelemetry-java/blob/master/api/src/main/java/io/opentelemetry/metrics/GaugeLong.java

Is it intended?